### PR TITLE
[docs][rbac-2] Update user-authz module documentation

### PR DESCRIPTION
**Pull request by z9r5**
## Description
Add documentation for the net role-based model.

Ref: #8538 

## Changelog entries

```changes
section: docs
type: chore
summary: Add documentation for the net role-based model.
impact_level:low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: modules/140-user-authz/docs/README_RU.md [hunk 1]
```diff
@@ -2,21 +2,114 @@
...
+{% alert level="warning" %}
+Документация модуля подразумевает использование [новой ролевой модели](#новая-ролевая-модель), если не указано иное.
+{% endalert %}
 
-- Управление доступом пользователей и групп на базе механизма RBAC Kubernetes.
+Модуль реализует ролевую модель доступа на базе стандартного механизма RBAC Kubernetes. Создает набор кластерных ролей (`ClusterRole`), подходящий для большинства задач по управлению доступом пользователей и групп.
+
+## Новая ролевая модель
+
+В отличие от [устаревшей ролевой модели](#устаревшая-ролевая-модель) DKP, новая ролевая модель не использует ресурсы `ClusterAuthorizationRule` и `AuthorizationRule`. Вся настройка прав доступа выполняется стандартным для RBAC Kubernetes путем: создание ресурсов `RoleBinding` или `ClusterRoleBinding`, с указанием в них необходимых ресурсов `ClusterRole`, из числа подготовленных модулем `user-authz`.
```
Comment (z9r5) [inline modules/140-user-authz/docs/README_RU.md:21]:
```suggestion
В отличие от [устаревшей ролевой модели](#устаревшая-ролевая-модель) DKP, новая ролевая модель не использует ресурсы `ClusterAuthorizationRule` и `AuthorizationRule`. Настройка прав доступа выполняется стандартным для RBAC Kuberenetes способом: с помощью создания ресурсов `RoleBinding` или `ClusterRoleBinding`, с указанием в них одной из подготовленных модулем `user-authz` роли.
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Pull request merged**